logbook-core/src/main/java/org/zalando/logbook/BodyFilters.java

```diff
@@ -50,13 +51,14 @@ public static BodyFilter accessToken() {
      * @return BodyFilter generated
      */
     @API(status = MAINTAINED)
-    public static BodyFilter replaceJsonStringProperty(final Set<String> properties, final String replacement) {
-        final String regex = properties.stream()
+    public static BodyFilter replaceJsonStringProperty(final Iterable<String> properties, final String replacement) {
+        final String regex = StreamSupport.stream(properties.spliterator(), false)
```

Comment:
Why are you using StreamSupport here?

Reply:
Or rather *Why are you switching from Set to Iterable?*

Reply:
Well, I always prefer to use `Iterable<T>` if that allows to do everything I want it to. 
It's more flexible for callers and it allows to keep method contract simple.

Reply:
That's is actually a trade of between semantics of `Set<T>` and flexibility of `Iterable<T>`.
Changes in method signature could be removed, if it makes no sense. Should I ?

Reply:
I believe it's a breaking API change anyway.

To me Set has the obvious benefit of highlighting to callers what I'm expecting here, i.e. no duplicates, since that wouldn't make any sense.